packages/ui/src/components/ChatMessage/ChatMessageActions/MessageActions.tsx

```diff
@@ -33,6 +33,16 @@ export default function MessageActions({
   const currentUserId = useCurrentUserId();
   const { addAttachment } = useAttachmentContext();
   const channel = useChannelContext();
+  const { data: group } = store.useGroup({ id: post.groupId ?? '' });
+  const currenUserIsAdmin = useMemo(
```

Comment:
We should really pull this out into something reusable. Looks like we do this same thing in 3 other places

Reply:
Agreed, will do that now.

Reply:
Wait, we *do* have a hook, we just don't use it consistently.

Reply:
Ok, fixed here and in those other places.

Reply:
Awesome, ty!